vendors/espressif/boards/esp32/aws_demos/config_files/iot_pkcs11_config.h

```diff
@@ -35,6 +35,16 @@
 /* A non-standard version of C_INITIALIZE should be used by this port. */
 /* #define pkcs11configC_INITIALIZE_ALT */
 
+/**
```

Comment:
Per gwicker comment, this config change will need to be added to the ESP32 test project as well.

Reply:
will do.